core/livepeernode.go

```diff
@@ -162,6 +162,7 @@ type LivepeerNode struct {
 type LivePipeline struct {
 	ControlPub  *trickle.TricklePublisher
 	StopControl func()
+	EventsPub   *trickle.TricklePublisher
```

Comment:
Shouldn't this be a sub? 🤔

Reply:
Hm I'm guessing this is orchestrator code? Or maybe it's unused? The O should start a server not a publisher :thonking: